api/ocm/tools/signing/digest.go

```diff
@@ -110,6 +111,15 @@ func NewRedirectedAccessMethod(m ocm.AccessMethod, bacc ocm.DataAccess) ocm.Acce
 	return &redirectedAccessMethod{m, bacc}
 }
 
```

Comment:
```suggestion
var _ accspeccpi.AccessMethodView = (*redirectedAccessMethod)(nil)
```

Ideally, you add this right below the `redirectedAccessMethod` type definition.

Reply:
done